pkg/riff/commands/function.go

```diff
@@ -114,7 +114,7 @@ func FunctionCreate(buildpackBuilder core.Builder, fcTool *core.Client) *cobra.C
 					return err
 				}
 			} else {
-				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "\nriff function create: built image %s for service %s.\n", f.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Image, f.Name)
+				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "\n%s function create: built image %s for service %s.\n", env.Cli.Name, f.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Image, f.Name)
```

Comment:
<nit>
Since this is going to be printed while running the `riff function create` command, I don't think the prefix is needed.
</nit>

Reply:
Agreed. We should only include the CLI command when we are giving the user a subsequent command they can run.